pkg/operator/petset.go

```diff
@@ -208,6 +219,12 @@ func makePetSetSpec(name, image, version, retention string, replicas int32, aler
 								MountPath: "/etc/prometheus/config",
 							},
 						},
+						Resources: v1.ResourceRequirements{
```

Comment:
Ah right, will fix.
We can bump it to 10. I intentionally didn't apply a limit because what knows why it might burst.